openedx/core/djangoapps/catalog/models.py

```diff
@@ -25,6 +27,14 @@ class CatalogIntegration(ConfigurationModel):
         )
     )
 
+    course_run_cache_ttl = models.PositiveIntegerField(
```

Comment:
@wajeeha-khalid @rlucioni 
As you know, I would much rather have LMS synchronize with the Catalog service asynchronously in a background process.  But we are trying to make [Wajeeha's PR](https://github.com/edx/edx-platform/pull/14234) build upon pre-existing work for our short-term needs.  Given this current short-term approach, I am fine with reusing the existing TTL for the course-sharing feature - and so this PR can be closed for now.  

However, once [Wajeeha's PR](https://github.com/edx/edx-platform/pull/14234) is deployed to Prod, we need to continue to monitor it and see whether different TTLs are needed, given the different SLAs for content-lifetime (i.e., the course-marketing URL is less likely to change as often as other course-catalog data).  FYI @edx/devops